response_operations_ui/contexts/reporting_units.py

```diff
@@ -0,0 +1,231 @@
+from flask import url_for
+
+CE_STATUS_CLASS = {
+    "Not started": "ons-status--info",
+    "In progress": "ons-status--pending",
+    "Completed": "ons-status--success",
+    "Completed by phone": "ons-status--success",
+    "No longer required": "ons-status--dead",
+}
+
+ENROLMENT_ENABLED = "ENABLED"
+ENROLMENT_PENDING = "PENDING"
+
+
+def build_reporting_units_context(
+    collection_exercises: list, ru: dict, survey: dict, respondents: list, case, unused_iac: str, permissions: dict
+) -> dict:
+    collection_exercise_section = _build_collection_exercise_section(
+        collection_exercises, ru["sampleUnitRef"], survey["shortName"], permissions["reporting_unit_edit"]
+    )
+    respondents_section = _build_respondents_section(
+        respondents,
+        case["id"],
+        collection_exercises[0]["id"],
+        collection_exercises[0]["tradingAs"],
+        ru["name"],
+        ru["sampleUnitRef"],
+        ru["id"],
+        survey["surveyRef"],
+        survey["shortName"],
+        survey["id"],
```

Comment:
Sorry my fault, I wasn't clear. If it's 1 or 2 elements in a list we should pass the value, more than that pass the list. It's a balance act between passing what it needs and how many vars for the function

Reply:
Ahh yeah that makes more sense - updated